cmd/metal-api/internal/service/ip-service.go

```diff
@@ -28,17 +30,23 @@ type ipResource struct {
 	webResource
 	ipamer ipam.IPAMer
 	mdc    mdm.Client
+	actor  *asyncActor
 }
 
 // NewIP returns a webservice for ip specific endpoints.
-func NewIP(ds *datastore.RethinkStore, ipamer ipam.IPAMer, mdc mdm.Client) *restful.WebService {
+func NewIP(ds *datastore.RethinkStore, ep *bus.Endpoints, ipamer ipam.IPAMer, mdc mdm.Client) *restful.WebService {
 	ir := ipResource{
 		webResource: webResource{
 			ds: ds,
 		},
 		ipamer: ipamer,
 		mdc:    mdc,
 	}
+	var err error
+	ir.actor, err = newAsyncActor(zapup.MustRootLogger(), ep, ds, ipamer)
+	if err != nil {
+		panic(err)
```

Comment:
Does this still work for creating the swagger spec through the CLI command?
Maybe also returning an error would be a little friendlier.